packages/client-sdk-web/src/internal/client-metadata-provider.ts

```diff
@@ -12,11 +13,13 @@ export interface ClientMetadataProps {
 export class ClientMetadataProvider {
```

Comment:
As of #528, the Web SDK uses `ClientMetadataProvider` to send headers because the interceptor didn't work.

However, this means the Web SDK headers are sent once per new client (e.g. cache, topic, auth, etc) rather than once at the start of any client usage, like with the NodeJS SDK.